IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricModelResource/Entities/IfcSectionedSurface/Documentation.md

```diff
@@ -1,15 +1,16 @@
-A kind of surface constructed by sweeping potentially varying open cross sections along a curve horizontally (or near horizontally). The surface is generated by sweeping the CrossSections between CrossSectionPositions; linear interpolation is assumed, unless transitions curves between cross section points are indicated by OpenCrossProfileDef.Tags.
-The CrossSections are oriented with the Y axis of each profile facing upwards in +Z direction or vertically perpendicular to the Directrix, depending on the Attribute FixedAxisVertical value.
+A surface constructed by sweeping potentially varying open cross sections along a curve horizontally (or near horizontally). The surface is generated by sweeping the _CrossSections_ between _CrossSectionPositions_; linear interpolation is assumed, unless transitions curves between cross section points are indicated by OpenCrossProfileDef.Tags.
+
+The profile normal is derived from the associated _IfcAxis2PlacementLinear_, not necessarily the tangent of the _Directrix_. The profile X axis is oriented perpendicularly to the left of the _Directrix_ (same direction as _RefDirection_ derived in _IfcAxis2PlacementLinear_) as facing forward along the directrix, and the profile Y axis is oriented upwards according to _Axis_ attribute of _IfcAxis2PlacementLinear_.
+
+Figure 1 illustrates an _IfcSectionedSurface_ defined using _IfcOpenCrossProfileDef_
+
 !["sectioned surfacequot;](../../../../../../figures/IfcSectionedSurface_1.png "Figure 1 &mdash; Sectioned surface using _IfcOpenCrossProfileDef_")
 
-In case of branching longitudinal breaklines, the SectionedSurface may use OpenCrossProfileDef instances with varying number of cross section points. In that case, the point in two consecutive cross sections that are connected are identified by the same tag value.
-!["sectioned surfacequot;](../../../../../../figures/IfcSectionedSurface_2.png "Figure 1 &mdash; Sectioned surface with branching longitudinal breaklines")
+In case of branching longitudinal breaklines, the _SectionedSurface_ may use _OpenCrossProfileDef_ instances with varying number of cross section points. In that case, the point in two consecutive cross sections that are connected are identified by the same tag value.
```

Comment:
We need stronger wording here. Two questions:
1. Would a WR make sense? Restricting the profile type.
2. Would we allow `IfcArbitraryOpenCrossProfileDef` with `IfcCartesianPointList` geometry?
```suggestion
In case of branching longitudinal breaklines, the _IfcSectionedSurface_ shall use _IfcOpenCrossProfileDef_ instances with varying number of cross section points. In that case, the point in two consecutive cross sections that are connected are identified by the same tag value.
```

Reply:
Solution:
Place in General usage the restrictions on profiles and Cartesian Point lists for Tags.

Reply:
Agree to move this to the usages.